frontend/cypress/integration/featureFiles/kiali_user.feature

```diff
@@ -0,0 +1,11 @@
+Feature: Kiali user
+
+  User wants to verify login user
+
+  Scenario: locate user on kiali page and verify
+    Given user opens base url
+    And user clicks Log In With OpenShift
+    And user clicks my_htpasswd_provider
+    And user fill in username and password
+    And user see console in URL
```

Comment:
please refactor step `Given user opens base url` with new step `Given user is at administrator perspective` 
for correct usage see
https://github.com/kiali/kiali/blob/bd662370f0950963d1151825915fc9e91db39474/frontend/cypress/integration/featureFiles/kiali_cookie.feature#L6